src/wallet/rpcwallet.cpp

```diff
@@ -2823,8 +2827,17 @@ UniValue bumpfee(const JSONRPCRequest& request)
         // new fee rate must be at least old rate + minimum incremental relay rate
         if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK()) {
             nNewFeeRate = CFeeRate(nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK());
+            if (specifiedConfirmTarget || payTxFee.GetFeePerK() == 0) {
```

Comment:
I think we want this logic to apply even if the user didn't specify a confirmation target -- just as long as paytxfee is 0.